code/__defines/observ.dm

```diff
@@ -0,0 +1,2 @@
+#define RAISE_EVENT(OBS, args...) var/decl/observ/__event = GET_DECL(OBS); __event?.raise_event(args);
```

Comment:
Can probably do `GET_DECL(OBS)?.raise_event(args)`, or even without the `?`, no? I don't like that pattern, but this is one of the few places it would make sense.